modules/rest-util/test/blaze/middleware/fhir/output_test.clj

```diff
@@ -205,5 +227,47 @@
       [:headers "Content-Type"] := "application/fhir+xml;charset=utf-8"
       [:body parse-xml :issue 0 :diagnostics] := "Invalid white space character (0x1e) in text to output (in xml 1.1, could output as a character entity)")))
 
+(deftest binary-resource-test
+  (testing "possible accept headers"
+    (are [accept content-type body]
```

Comment:
please don't combine `given` with `are` because `given` already uses `is` inside. it's better to use `doseq` together with `given`

Reply:
Should we also rewrite the existing tests that use `are` to use `is`?